### ISSUE TITLE
feature request: angproduct

### ISSUE THREAD

**Issue by ArTourter** [josephwright/siunitx#580]
We have `\numproduct` and `\qtyproduct` but no `\angproduct`.

I have a use case for it when describing climate model resolutions latitude x longitude angular res. I have created my own shortcut but would be useful to have this part of this package.

**Comment by josephwright** [comment]
What I guess I'm not seeing immediately is what the 'combining advantage' is. With `\qtyproduct` and even `\numproduct` there are some common elements that can get pulled out, e.g. exponent. What's the case with angles?

**Comment by ArTourter** [comment]
yes from that point of view, I see what you mean. I had not looked at it that way.  I was thinking from the consistency and convenience pov as I tried to typeset  number products, qty products and angle product in the same document.
